client/landing/stepper/declarative-flow/onboarding.ts

```diff
@@ -113,9 +119,9 @@ const onboarding: Flow = {
 					if ( ! cartItems?.[ 0 ] ) {
 						// Since we're removing the paid domain, it means that the user chose to continue
 						// with a free domain. Because signupDomainOrigin should reflect the last domain
-						// selection status before they land on the checkout page, we switch the value
-						// to "free".
-						setSignupDomainOrigin( SIGNUP_DOMAIN_ORIGIN.FREE );
+						// selection status before they land on the checkout page, this value can be
+						// 'free' or 'choose-later'
```

Comment:
c/c @escapemanuele 
I think this is better, right?

Reply:
Sounds cool 👍